kikar_hamedina/core/templates/core/facebook_status.html

```diff
@@ -88,14 +88,14 @@
                                      style="text-align: center">
                                     {% if  status.attachment.is_youtube_video %}
                                         <object data="{{ status.attachment.source }}"  >
-                                            <embed>
+                                            <embed autoplay="false">
                                                 <source src="{{ status.attachment.source }}" type="video/mp4">
                                                 <source src="{{ status.attachment.source }}" type="video/ogg">
                                             {% trans 'Apologies, your broswer does not support displaying videos.' %}
                                             </embed>
                                         </object>
                                     {% else %}
-                                     <video width="auto" height="auto" autoplay controls>
+                                     <video width="auto" height="auto">
```

Comment:
This is a mistake. it is not relevant. pls do not update the main repository with this file.